cvat-ui/src/utils/controls-logger.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { getCore } from 'cvat-core-wrapper';
+import { LogType } from 'cvat-logger';
+import config from 'config';
+
+const core = getCore();
+const { CONTROLS_LOGS_INTERVAL } = config;
+
+const classFilter = ['ant-btn'];
+const parentClassFilter = ['ant-btn'];
+
+class EventRecorder {
+    #savingInterval: number | null;
+    public constructor() {
+        this.#savingInterval = null;
+    }
+
+    public log(event: MouseEvent): void {
+        const { target } = event;
+        if (!target) {
+            return;
+        }
+
+        let element = (target as HTMLElement);
+        let toRecord = this.isEventToBeRecorded(element, classFilter);
+
+        const logData = {
+            text: element.innerText,
+            classes: element.className,
```

Comment:
Need to think what kind of information we are going to record about pressed buttons. Maybe  need to add descriptive class name to all the buttons. Now not everything has it and it is not always clear what button has been pressed.